client/modules/Preview/EmbedFrame.jsx

```diff
@@ -314,13 +314,50 @@ function EmbedFrame({ files, isPlaying, basePath, gridOutput, textOutput }) {
 
   useEffect(renderSketch, [files, isPlaying]);
   return (
-    <Frame
-      aria-label="Sketch Preview"
-      role="main"
-      frameBorder="0"
-      ref={iframe}
-    />
+    <div style={{ position: "relative" }}>
+      {/* Expand Button */}
+      <button
+        onClick={() => {
+          if (iframe.current.style.width === "100vw") {
```

Comment:
As a note, styling should usually be edited from the respective `scss` or `css` file, and not within the component.